pwndbg/gdblib/kernel/__init__.py

```diff
@@ -76,6 +78,21 @@ def kversion() -> str:
     return pwndbg.gdblib.memory.string(version_addr).decode("ascii").strip()
 
 
+@requires_debug_syms()
+@pwndbg.lib.memoize.reset_on_start
+def krelease() -> Tuple[int, ...]:
+    # try to extract (major.minor.patch) version
+    match = re.search(r"Linux version (\d+)\.(\d+)\.(\d+)", kversion())
+    if match:
+        return tuple(map(int, match.groups()))
+    # try to extract (major.minor) version and append 0 for patch version
+    match = re.search(r"Linux version (\d+)\.(\d+)", kversion())
```

Comment:
Instead of doing the regex search twice, I think we can do it once with something like this:
```
match = re.search(r"Linux version (\d+)\.(\d+)(?:\.(\d+))?", kversion())
```

For 'Linux version 5.10' `match.groups()` returns `('5', '10', None)` and for 'Linux version 5.10.108'  it returns `('5', '10', '108')`

Reply:
i revised the function to use only one regex now and made the patch version in the tuple optional. i do not really see a reason to require it anyway as pythons tuple comparison also works on tuples of different lengths